content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -309,6 +309,7 @@ Here's one scenario where you would set it. Suppose the Container listens on 127
 and the Pod's `hostNetwork` field is true. Then `host`, under `httpGet`, should be set
 to 127.0.0.1. If your pod relies on virtual hosts, which is probably the more common
 case, you should not use `host`, but rather set the `Host` header in `httpHeaders`.
+Also, you can not use service name as `host` param since `kubelet` run probe on the node.
```

Comment:
How about:
```markdown
kubelet will make the probe connection. If you use the service name in the `host` parameter,
kubelet will be unable to resolve it and the probe will fail.
```

Reply:
> kubelet will make the probe connection
How about `kubelet will make the probe connection at the node, not in pod. `

Reply:
It sounds like the points we want to emphasize are:
- probing is something that kubelet does (and not, for example, the control plane)
- kubelet only resolves names locally; it doesn't make sense to probe a cluster- or namespace- wide thing because we're interested in a specific pod on a specific node

Does that match what you want to convey here @daixiang0 ?

Reply:
yep, you are totally right.